docs/rackhd/ubuntu_package_installation.rst

```diff
@@ -49,7 +53,45 @@ Ensure Node.js is at version 4.x:
     $ node -v
     v4.4.5
 
-Install the prerequisite packages:
+
+Install & Basically Configure RackHD
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ **after Prerequisites installation, there're two options to install and configure RackHD from package**
+ Either (a) or (b) can lead the way to install RackHD from debian packages.
+
+   (a) Install/Configure with Ansible Playbook
+   (b) Install/Configure with Step by Step Guide
+
+
+(a) Install/Configure with Ansible Playbook
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+a.1. install git and ansible
+.. code::
+
+  sudo apt-get install  git
+  sudo apt-get install  ansible
+
+a.2. clone RackHD code
+
+.. code::
+
+  git clone https://github.com/RackHD/RackHD.git
+
+a.3. Run the ansible playbooks, it will install the prerequisites packages, install RackHD debian packages, copy default config files..
```

Comment:
You're sort of mixing a title line with content here. This might be better as:

`a.3. Run the ansible playbooks`

`These will install the prerequisite packages, install the RackHD debian packages, and copy default files.`